backend/src/data/migrations/20220818224217_add_fullname_to_user_details.ts

```diff
@@ -0,0 +1,38 @@
+import { Knex } from 'knex';
+
+enum ColumnName {
+  FULL_NAME = 'full_name',
+  FIRST_NAME = 'first_name',
+  LAST_NAME = 'last_name',
+}
+
+enum TableName {
+  USERS = 'users',
+  USER_DETAILS = 'user_details',
+}
+
+async function up(knex: Knex): Promise<void> {
+  await knex.schema.table(TableName.USER_DETAILS, (table) => {
+    table.string(ColumnName.FULL_NAME).notNullable();
+    table.dropColumn(ColumnName.FIRST_NAME);
```

Comment:
we need to move all user-name from old table to a new one

Reply:
@what1s1ove so I need to copy all existing fullNames from user table to userDetails, and delete them from the user table, correct? could you please give me a hint on how can I do it?

Reply:
write to the internal chat, please